index.html

```diff
@@ -65,13 +70,11 @@ <h3 class="connected">
     </section>
 
     <section class="middle" id="portfolio">
-      <ul class="first-item">
-
+      <ul class="first-item" id="first-item">
         <li class="port-1"><img src="./Graphics/Snapshoot Portfolio.jpg" alt="portfolio-image-1"></li>
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=You%20need%20to%20update%20the%20main%20page%20so%20that%20the%20projects%20section%20is%20created%20dynamically%20using%20the%20information%20stored%20in%20that%20JavaScript%20object.), Please make sure that all projects section is generated dynamically using javascript and all hard code for them in `index.html` file is removed.